src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -18,8 +19,7 @@ object AstTools {
     def findTerminals(): Seq[Terminal] = AstTools.findTerminals(astNode)
     def findTopLevelMemberAccesses(): Iterable[Ast] = AstTools.findTopLevelMemberAccesses(astNode)
     def sourceString(): String = astNode.asInstanceOf[Terminal].getSourceString
-    
-    def wdlType: WdlType = {
+    def wdlType(wdlSyntaxErrorFormatter: WdlSyntaxErrorFormatter): WdlType = {
```

Comment:
Yeah I do kinda see what you mean... though if I wrote it this way because I preferred it to making it an implicit parameter, though perhaps there's a better way to organize this whole thing